.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Changes
+
+<!--
+  Required.
+
+  Briefly describe the changes introduced by this pull request.
+  Also link relevant issues with the "closes", "fixes" or "resolves" keywords,
+  and add co-authors where appropriate with the "co-authored-by" keyword.
+
+  Example:
+  Implemented the functionality to update the user's name.
```

Comment:
This information could be directly the title?

Reply:
If it is very short then yes, since we're mostly using "squash and merge" for PRs and the title ends up being the commit message.

But here is a good place to be a little bit more descriptive to help the code reviews.